src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public double areaCalculator() {
+        return 0;
+    }
+
+    @Override
+    public abstract void draw();
```

Comment:
```suggestion
```
this class is abstract, no need to repeat methods from interfaces
you should create another interface AreaCalculator with a method getArea()